Firmware/Configuration.h

```diff
@@ -43,7 +42,9 @@ extern const char _sPrinterMmuName[] PROGMEM;
 #define FW_COMMIT_HASH_LENGTH 1
 #define FW_COMMIT_HASH "0"
 #define FW_REPOSITORY "Unknown"
-#define FW_VERSION_FULL FW_VERSION "-unknown"
+#ifndef FW_VERSION_FULL
+#define FW_VERSION_FULL FW_VERSION
```

Comment:
This entire ifndef can be removed. it's impossible for FW_VERSION_FULL to not be defined in the if block above.